### PR TITLE
feat: flag exposure in personal dashboard

### PR DESCRIPTION
**Pull request by kwasniew**
## About the changes

Show last seen widget and lifecycle widget when selecting a flag
Screenshot 2024-09-25 at 10 31 30

Same last seen as the project flags and single flag view
Screenshot 2024-09-25 at 10 31 34

Same lifecycle as the project flags and single flag view
Screenshot 2024-09-25 at 10 31 39

Details:
* create a new FlagExposure component to encapsulate exposure info
* had to improve lifecycle tooltip so that they don't reach to the URL for the project id. Instead we pass it explicitly from the parent. The reason for that is that personal dashboard is not tied to a project based URL 
* when archiving the flag from the lifecycle widget you end up on the same screen but we need to refresh the personal dashboard data. In other places we usually navigate to the project flags view after the archive but here it makes sense to stay in the dashboard
* if you archive a feature from the lifecycle tooltip with change requests enabled you get a change request created but you will only see it when you go to the project with a change request. Change request bar is tied to the project view. I think it's such a rare corner case that it's ok for now

### CODE REVIEW

File: frontend/src/component/feature/FeatureView/FeatureOverview/FeatureLifecycle/FeatureLifecycleTooltip.test.tsx [hunk 1]
```diff
@@ -23,24 +22,17 @@ const renderOpenTooltip = (
...
-        </Routes>,
+        <FeatureLifecycleTooltip
+            stage={stage}
+            onArchive={onArchive}
+            onComplete={onComplete}
+            onUncomplete={onUncomplete}
+            loading={loading}
+            project={'default'}
+        >
+            <span>child</span>
```
Comment (thomasheartman) [inline frontend/src/component/feature/FeatureView/FeatureOverview/FeatureLifecycle/FeatureLifecycleTooltip.test.tsx:33]: curious: what is this child? is it just the text child in a span?

File: frontend/src/component/feature/FeatureView/FeatureOverview/FeatureLifecycle/FlagExposure.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+            )}
+
+            {feature.project ? (
+                <MarkCompletedDialogue
+                    isOpen={showMarkCompletedDialogue}
+                    setIsOpen={setShowMarkCompletedDialogue}
+                    projectId={feature.project}
+                    featureId={feature.name}
+                    onComplete={refetchFeature}
+                />
```
Comment (thomasheartman) [inline frontend/src/component/feature/FeatureView/FeatureOverview/FeatureLifecycle/FlagExposure.tsx:68]: Can a feature not have a project?

**Review by thomasheartman — APPROVED** [review]

File: frontend/src/component/feature/FeatureView/FeatureOverview/FeatureLifecycle/FeatureLifecycleTooltip.test.tsx (hunk 1 above)
Comment (kwasniew) [reply frontend/src/component/feature/FeatureView/FeatureOverview/FeatureLifecycle/FeatureLifecycleTooltip.test.tsx:33]: this component accepts any ReactElement as children prop so wanted to specify sth that doesn't look like string. In tests we find this element by text so passing child without a span would also work

File: frontend/src/component/feature/FeatureView/FeatureOverview/FeatureLifecycle/FlagExposure.tsx (hunk 2 above)
Comment (kwasniew) [reply frontend/src/component/feature/FeatureView/FeatureOverview/FeatureLifecycle/FlagExposure.tsx:68]: Yes, a placeholder feature has an empty project
